services/src/main/scala/cromwell/services/metadata/impl/deleter/DeleteMetadataConfig.scala

```diff
@@ -0,0 +1,26 @@
+package cromwell.services.metadata.impl.deleter
+
+import com.typesafe.config.Config
+import common.Checked
+import common.validation.Validation._
+import net.ceedubs.ficus.Ficus._
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.util.Try
+
+final case class DeleteMetadataConfig(nonSuccessInterval: FiniteDuration,
+                                      delayAfterWorkflowCompletion: FiniteDuration)
+
+object DeleteMetadataConfig {
+
+  def parseConfig(archiveMetadataConfig: Config): Checked[DeleteMetadataConfig] = {
+    val defaultNonSuccessInterval: FiniteDuration = 1 minute
+    val defaultDelayAfterWorkflowCompletion: FiniteDuration = 10 minutes
```

Comment:
Isn't this far too soon as a default? More like a year or 400 days as I had written in my other PR?

Reply:
👍  and in fact, on consideration, this actually feels like something that doesn't even deserve a default. If you're specifying a delete block, you probably should be forced to specify your delay explicitly.